### DISCUSSION TITLE
Shortcut for "Grab Color"

### DISCUSSION THREAD

**Discussion by NucAsm** [flameshot-org/flameshot#3366] [Ideas]
![image](https://github.com/flameshot-org/flameshot/assets/146146235/475cb00d-f65e-492a-b3b5-df54b1aa7add)
Im not great at coding, infact i dont know wth im doing most of the time. 
It could be that this is already implemented somewhere, to lazy to go over the code.

Though i haven't found a way to set a shortcut for the color grab feature.
It'd be great to have since painting over a picture looks alot smoother if the part your trying to either hide or add conforms with the local colors of the section within the screenshot you are working with

Feature is handy. 
To bad im stretching my cursor across the screen in order to grab another color template to the picture that im working with.

**Comment by mmahmoudian** [comment]
I believe this is a good idea. I have a feeling that I have seen this before, but for some reason I cannot find it.

Would you please create a Feature request using the following link, and copy-paste this to there? As maintainer of this repo I can create an issue from this discussion, but the created issue will be under my name and not yours. I would rather keep you as the person who made the suggestion. 

https://github.com/flameshot-org/flameshot/issues/new/choose

Thanks.
